sdk/azidentity/client_assertion_credential_test.go

```diff
@@ -77,3 +78,37 @@ func TestClientAssertionCredentialCallbackError(t *testing.T) {
 		t.Fatalf(`unexpected error: "%v"`, err)
 	}
 }
+
+func TestClientAssertionCredential_Live(t *testing.T) {
```

Comment:
Curious why this one can't be recorded.

Reply:
It's recorded. The recording doesn't appear in this PR because it's a new test. I pushed the recording to the assets repo after the initial migration.